modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/BaseLayout/index.js

```diff
@@ -31,26 +31,48 @@ const PAGE_SKELETON_LAYOUT = {
 
 const Layout = () => {
 	const location = useLocation();
+	const [, ...currentPath] = location.pathname?.split('/')?.filter(Boolean);
 
 	const [
 		{project, sessionId, subscriptionGroups, userAccount},
 	] = useCustomerPortal();
 
 	const getCurrentPage = useCallback(() => {
-		const [, ...currentPath] = location.pathname.split('/').filter(Boolean);
-
 		return currentPath.length
 			? currentPath.slice(-1)[0]
 			: PAGE_TYPES.overview;
-	}, [location]);
+	}, [currentPath]);
+
+	const getCurrentProduct = () => {
+		const activationKey = 'activation';
+
+		const isProduct = !!currentPath?.find((path) => path === activationKey);
```

Comment:
I think you can use ".some" instead "!!.find"

Reply:
Nice catch

Reply:
@joanagrego

Reply:
@joanagrego

Reply:
Nice! Done!